ci-operator/step-registry/openstack/provision/sriov-operator/openstack-provision-sriov-operator-commands.sh

```diff
@@ -62,8 +62,6 @@ if [ -n "${is_dev_version:-}" ]; then
     echo "The SR-IOV will be installed from Github using release-${oc_version} branch."
     git clone --branch release-${oc_version} https://github.com/openshift/sriov-network-operator /tmp/sriov-network-operator
     pushd /tmp/sriov-network-operator
-    # Until https://github.com/openshift/sriov-network-operator/pull/613 merges
-    cp manifests/stable/supported-nic-ids_v1_configmap.yaml deploy/configmap.yaml
```

Comment:
That seems fine, we have the `Red_Hat_Virtio_network_device: "1af4 1000 1000"` NIC we need in https://github.com/openshift/sriov-network-operator/blob/release-4.12/deploy/configmap.yaml